docs/VALIDATORS.md

```diff
@@ -180,6 +180,7 @@
   * [NfeAccessKey](NfeAccessKey.md)
   * [NotBlank](NotBlank.md)
   * [NotOptional](NotOptional.md)
+  * [Pesel](Pesel.md)
```

Comment:
Hey, @regdos, first of all, I'm sorry by my late.

Looks like you just forgot to add this file to your commit, could add it to this pull request?